library/blinkt.py

```diff
@@ -55,4 +55,14 @@ def set_pixel(x, r, g, b, brightness=None):
         brightness = int(31.0 * brightness) & 0b11111
     pixels[x] = [int(r) & 0xff,int(g) & 0xff,int(b) & 0xff,brightness]
 
+def set_all_pixel(r, g, b, brightness=None):
+    if brightness is None:
+        for x in range(8):
+            brightness = pixels[x][3]
+            pixels[x] = [int(r) & 0xff,int(g) & 0xff,int(b) & 0xff,brightness]
```

Comment:
Could possibly be simplified to:

```
pixels[x][0:3] = [int(n) & 0xff for n in [r, g. b]]
```

The `[0:3]` means that only the first 3 items of the list will be replaced, so brightness will be untouched as intended.

The list comprehension is an concise way to apply the same cast to int and range clamping to r, g and b without repetition.

Otherwise, good stuff! And a useful method to have.